frontend/front/src/components/CustomSnackbar.js

```diff
@@ -0,0 +1,13 @@
+import { Snackbar, Alert } from "@mui/material";
+
+function CustomSnackbar({ open, onClose, message, severity }) {
```

Comment:
Hi, one more suggestion: maybe you can provide a standard value for the props as well. This way, if someone forgets to use one of these props, they will have a default value. Let me know if you understand!

Reply:
I will be creating a new PR for this remove-create-survey-button off of the main branch, this one is not correlated to the previous PR